docs/backend/subscribers.md

```diff
@@ -11,8 +11,268 @@ myst:
+
+
+### Register an event handler on content type creation
+
+The following example demonstrates how to register an event handler when a content type is created.
+
+In your {file}`.product/your/product/configure.zcml` insert the following code.
+
+{lineno-start=1}
+```xml
+<subscriber
+    for=".interfaces.IMyContentTypeClass
+         zope.lifecycleevent.IObjectCreatedEvent"
+    handler=".your_python_file.your_method"
+    />
+```
+
+The second line defines to which interface you want to bind the execution of your code. 
+Here, the event handler code will only be executed if the object is a content type providing the interface `.interfaces.IMyContentTypeClass`.
+If you want this to be interface agnostic, insert an asterix `*` as a wildcard instead.
+
+The third line defines the event on which this should happen, which is `IObjectCreatedEvent`. 
+For more available possible events to use as a trigger, see {ref}`subscribers-event-handlers`. 
+
+The fourth line gives the path to the callable function to be executed.
+
+Create your {file}`.product/your/product/your_python_file.py` and insert the following code.
+
+```python
+def your_subscriber(object, event):
+    # do something with your created content type
+```
+
+
+### Subscribe to an event using ZCML
+
+Subscribe to a global event using {term}`ZCML` by inserting the following code in your {file}`.product/your/product/configure.zcml`.
+
+```xml
+<subscriber
+    for="Products.PlonePAS.events.UserLoggedOutEvent"
+    handler=".smartcard.clear_extra_cookies_on_logout"
+    />
+```
+
+For this event, the Python code in {file}`smartcard.py` would be the following.
+
+```python
+def clear_extra_cookies_on_logout(event):
+    # What event contains depends on the
+    # triggerer of the event and event class
+    request = event.object.REQUEST
+```
+
+The following example for a custom event subscribes content types to all `IMyEvents` when fired by `IMyObject`.
+
+```xml
+<subscriber
+    for=".interfaces.IMyObject
+         .interfaces.IMyEvent"
+    handler=".content.MyObject.myEventHandler"
+    />
+```
+
+The following example shows how to subscribe a content type to the life cycle event.
+
+```xml
+<subscriber
+    zcml:condition="installed zope.lifecycleevent"
+    for=".interfaces.ISitsPatient
+         zope.lifecycleevent.IObjectModifiedEvent"
+    handler=".content.SitsPatient.objectModified"
+    />
+```
+
+
+### Subscribe to an event using Python
+
+The following example shows how subscription is valid through the process life cycle.
+In unit tests, it is important to clear test event handlers between the test steps.
+
+```python
+import zope.component
+
+def my_event_handler(context, event):
+    """
+    @param context: Zope object for which the event was fired. Usually this is a Plone content object.
+
+    @param event: Subclass of event.
+    """
+    pass
+
+gsm = zope.component.getGlobalSiteManager()
+gsm.registerHandler(my_event_handler, (IMyObject,IMyEvent))
+```
```

Comment:
I'd suggest to remove this section. We should go with "registering in zcml" as the default in documentation. Otherwise its maybe more confusing than helping ...